src/lib/components/globalhealthbar/GlobalHealthBar.component.js

```diff
@@ -0,0 +1,139 @@
+//@flow
+import React from 'react';
+import VegaChart from '../vegachart/VegaChart.component.js';
+
+type Props = {
+  id: string,
+  alertSegments: {
+    severity: 'critical' | 'warning' | 'none',
+    startSegment: string,
+    endSegment: string,
+    description: string,
+  }[],
+  start: { date: number, month: number, year: number },
+  end: { date: number, month: number, year: number },
+  height?: number,
+  width?: number,
+  theme?: 'light' | 'dark' | 'custom',
```

Comment:
It should be aware of the currently selected theme using useTheme hook, so I think we can drop this theme